### PR TITLE
Change cron timings for 4.17 PowerVS jobs

### PR DESCRIPTION
**Pull request by Shilpa-Gokul**

### CODE REVIEW

File: ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml [hunk 1]
```diff
@@ -577,7 +577,7 @@ tests:
     workflow: openshift-e2e-libvirt-upi
 - as: ocp-e2e-ovn-ppc64le-powervs-original
   cluster: build01
-  cron: 0 0,6,12,18 * * *
+  cron: 0 12 * * *
```
Comment (tvardema) [inline ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml:580]: Do we want this job running at the same time as 4.15?

Comment (mjturek) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml:580]: Also, historically we've used the same timings across different regions and it's worked fine!

Comment (tvardema) [reply ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.16.yaml:580]: Perfect!  Just checking :D

**Comment by hamzy** [comment]
BTW Mike, if you expand the "Interacting with pj-rehearse" twisty, it tells you the format of the command :)

**Pull request merged**